src/features/pricing/additionalServicePricing/__tests__/AdditionalServicePricing.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import AdditionalServicePricing, { AdditionalServicePricingProps } from '../AdditionalServicePricing';
+import { data } from '../__fixtures__/data';
+
+describe('AdditionalServicePricing', () => {
+  const initialProps: AdditionalServicePricingProps = { data, loading: false, openModal: jest.fn() };
+
+  const getWrapper = (props: Partial<AdditionalServicePricingProps> = {}) =>
+    shallow(<AdditionalServicePricing {...initialProps} {...props} />);
+
+  beforeEach(() => {
```

Comment:
No mocks to reset here either? Or am I missing something 👀

Reply:
Same here, there's `openModal: jest.fn()` In the `initialProps`.
I added tests for edit buttons that call this function.